src/ol/source/vector.js

```diff
@@ -610,11 +610,12 @@ ol.source.Vector.prototype.getClosestFeatureToCoordinate = function(coordinate,
  *
  * This method is not available when the source is configured with
  * `useSpatialIndex` set to `false`.
+ * @param {ol.Extent=} opt_extent Destination extent.
```

Comment:
We expose this `opt_extent` in almost every `ol.extent.*` function but it's also very poorly documented there.